packages/slate-commons/src/commands/isEmptyParagraphElement.ts

```diff
@@ -1,28 +1,18 @@
 import { isParagraphNode } from '@prezly/slate-types';
 import { Node } from 'slate';
-import type { Editor } from 'slate';
 
-import { isNodeEmpty } from './isNodeEmpty';
-
-export interface IsEmptyParagraphElementOptions {
-    shouldTrim?: boolean;
+interface Options {
+    ignoreWhitespace?: boolean;
```

Comment:
imho i am not sure about "ignoreWhitespace" name
i.e. in this example "Hello world" has whitespace in the middle, but it will not be ignored

Reply:
Hm... Good point. I didn't think of this.

Let's call it just `trim` then?

Reply:
> imho i am not sure about "ignoreWhitespace" name
i.e. in this example "Hello world" has whitespace in the middle, but it will not be ignored

After thinking over night, I think `ignoreWhitespace` could still do the job, as it has no effect on your `Hello world` counter-example whether we *trim* whitespace or ignore all of them (`.replace(/s/g, '')`) -- the string is still not empty (`Helloworld`). So the difference is effectively indistinguishable for the use-case :shrug: :) 

But let's keep it `trim`. Makes more sense to have a shorter name.